docs/build-general.md

```diff
@@ -1,8 +1,7 @@
 # Building LibVCX
 
 ### 1. Build dependencies
-In order to build and call LibVCX code, you need to first build Libindy and Libnullpay. To 
-build these them follow instructions for your platform [here](https://github.com/hyperledger/indy-sdk/tree/master/docs/build-guides).
+In order to build and call LibVCX code, you need to first build [VdrTools](https://gitlab.com/evernym/verity/vdr-tools).
```

Comment:
I would unify the spelling in this document to `libvcx` and `vdr-tools`

Reply:
udpated